readme.md

```diff
@@ -0,0 +1,23 @@
+# [processhacker](https://github.com/processhacker)/[plugins-extra](https://github.com/processhacker/plugins-extra)
+
+These are small plugins that are not included with Process Hacker by default.
+
+## Compilation
+_Quoting https://wj32.org/processhacker/forums/viewtopic.php?t=2801#p9194_
```

Comment:
These instructions would be the authoritative source so linking elsewhere like a copy/paste should be removed.

Reply:
```suggestion
```

Changes are easy enough to do, so I'll try to use the new "Suggested Changes" feature!